js/script.js

```diff
@@ -354,10 +354,10 @@ $(document).ready(function(){
 					datasets : [
 						{
 							backgroundColor: [
-				                OCA.Theming.color,
-				                'rgb(249, 249, 249)',
-				            ],
-				            borderWidth: 0,
+								OCA.Theming.color,
```

Comment:
I just realised that theming can be disabled, so you'll have to check if OCA.Theming exists first and add a fallback color ;)

Reply:
really? you mean disabling the theming app? How do you suggest to check for that in the correct way?

Reply:
> you mean disabling the theming app?

yes :)
I would just do a ternary check:
```js 
OCA.Theming
    ? OCA.Theming.color
    : 'rgb(54, 129, 195)'
```

Reply:
ahh cool. this makes sense